doc/releases/release-notes-2.3.rst

```diff
@@ -500,7 +500,23 @@ Bluetooth
 Build and Infrastructure
 ************************
 
-* <TBD>
+* Zephyr CMake package
+
+  * Zephyr repository now includes a Zephyr CMake package.
+    This allows for registrering Zephyr in the CMake user package registry and
+    allows for easier integration into Zephyr applications, by using the CMake
+    function, ``find_package(Zephyr ...)``
+    Registrering the Zephyr CMake package in the CMake user package registry
+    removes the need for setting of ``ZEPHYR_BASE``, ``source zephyr-env.sh``,
```

Comment:
```suggestion
    removes the need for setting of ``ZEPHYR_BASE``, sourcing ``zephyr-env.sh``,
```

Reply:
Isn't it too much text for release notes?

Reply:
Maybe, but i'll let other decide if this should be reduced.

But the reason for making a bit longer entry on this in the release notes is because this change is impacting everyone creating applications based on Zephyr.
The `find_package(Zephyr)` is now a fundamental part of the build system.

Compared to, as example, changes to a driver, that could be impacting fewer users.

Reply:
I think this makes sense. Some people will only ever read the release notes, so highlighting this here is important for them.